src/web_ui/package.json

```diff
@@ -11,6 +11,8 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "clsx": "^1.1.0",
+    "enzyme": "^3.11.0",
```

Comment:
`React testing library` is already loaded so the enzyme import adds unnecessary dependencies to the project